### PR TITLE
show info user

### PR DESCRIPTION
**Pull request by at-betruong**
## Description
-show info user
-update info user
## Reference Requirements
- Task ID: TI-1

## Reference Isuees
#{issue_id}

## Migrations
YES | NO

## Impacted Areas in Application
List general components of the application that this PR will affect:

-

## SQL query and `explain`
List of all queries and attach the result of each explaining query (in text, screenshot...)

- Refer to [Using Explain with MySQL](https://viblo.asia/p/su-dung-explain-de-toi-uu-cau-lenh-mysql-BYjv44gmvxpV)

### CODE REVIEW

File: routes/web.php [hunk 1]
```diff
@@ -29,6 +29,7 @@
 });
 Route::group(['as' => 'user.', 'namespace' => 'User'], function () {
     Route::get('/', 'HomeController@index');
+    Route::get('showinfor', 'HomeController@profile')->name('showinfor');
```
Comment (at-xinhnguyen) [inline routes/web.php]: nếu đã đăng nhập thì chỗ này nên là `users/{id}/profile` name:`profile`

File: routes/api.php [hunk 2]
```diff
@@ -22,5 +22,7 @@
     Route::post('register', 'LoginController@register')->name('register');
     Route::group(['middleware' => 'auth:api'], function () {
         Route::post('logout', 'LoginController@logout');
+        Route::put('users/profile', 'UserController@update');
```
Comment (at-xinhnguyen) [inline routes/api.php]: `users/id/profile`

File: routes/web.php [hunk 3]
```diff
@@ -29,6 +29,7 @@
 });
 Route::group(['as' => 'user.', 'namespace' => 'User'], function () {
     Route::get('/', 'HomeController@index');
+    Route::get('showinfor', 'HomeController@info')->name('showinfor');
```
Comment (at-xinhnguyen) [inline routes/web.php]: showinfor => profile
HomeController@info -> UserController@getProfile
name => user.profile

**Review by ducxinh — APPROVED** [review]
LGTM

**Review by ghost — APPROVED** [review]
LGTM

**Pull request merged**